pkg/reconcilermanager/constants.go

```diff
@@ -136,8 +136,14 @@ const (
 	// HelmValues is the OS env variable key for the Helm chart values.
 	HelmValues = "HELM_VALUES"
 
-	//HelmValuesFiles is the OS env variable key for Helm values files.
-	HelmValuesFiles = "HELM_VALUES_FILES"
+	// HelmConfigMapRef is the path where valuesFiles from ConfigMaps are mounted.
+	HelmConfigMapRef = "HELM_CONFIGMAP_REF"
+
+	// HelmValuesFrom is the OS env variable key for the names of ConfigMaps holding valuesFiles.
+	HelmValuesFrom = "HELM_VALUES_FROM"
+
+	// HelmValuesKeyMergeMode is the OS env variable key for Helm values files merge strategy.
+	HelmValuesKeyMergeMode = "HELM_VALUES_MERGE_MODE"
```

Comment:
nit: should it be HELM_VALUES_KEY_MERGE_MODE?

Reply:
Ah I missed that one. Thanks for the catch, I'll update that